gravitee-apim-console-webui/src/management/application/details/subscriptions/subscription/application-subscription.component.html

```diff
@@ -120,6 +120,14 @@
       ></subscription-api-keys
     >
   }
+
+  @if (pageVM.subscription?.plan?.security === undefined && !!pageVM.subscription?.configuration) {
```

Comment:
```suggestion
  @if (!pageVM.subscription?.plan?.security && !!pageVM.subscription?.configuration) {
```
WDYT in case security is `null`?

Reply:
why not 🚀